recipes/h5py/bld.bat

```diff
@@ -0,0 +1,5 @@
+"%PYTHON%" setup.py configure --hdf5="%LIBRARY_PREFIX%"
```

Comment:
I like that better too.

PS: I had to set `%LIBRARY_INC%` and `%LIBRARY_LIB%` for `netCDF4-python`.